doc/source/MANUAL.rst

```diff
@@ -352,6 +356,18 @@ method)::
 
     group_regex=([^\.]+\.)+
 
+Or, you can use the following option instead::
```

Comment:
How about instead of this we write:


```suggestion
However, because grouping tests at the class level is a common use case there is also a config option, ``parallel_class``, to do this. For example, you can use::

    parallel_class=True
    
and it will group tests in the same class together.
```